src/Wpf.Ui/Controls/MessageBox.cs

```diff
@@ -238,6 +237,18 @@ public void Show(string title, object content)
     //    base.OnContentChanged(oldContent, newContent);
     //}
 
+    private void SetWindowStartupLocation()
+    {
+        if (Application.Current.MainWindow != null)
```

Comment:
Can we do `Application.Current?.MainWindow` instead? The else branch doesn't require `Application.Current` to be set